sr/comp/knockout_scheduler/static_scheduler.py

```diff
@@ -34,6 +49,31 @@ class StaticMatchInfo(TypedDict):
     display_name: str | None
 
 
+def parse_team_ref(team_ref: str) -> tuple[int, int, int]:
+    """
+    Parse a string reference into round/match/position.
+
+    See docstring on `StaticMatchTeamReference` for further details -- this
+    function supports both the compressed and RMP formats.
+    """
+
+    if len(team_ref) == 3 and team_ref.isdecimal():
```

Comment:
`isdecimal` technically allows more than just `0-9` but is close enough (and previously we allowed a lot more than this!)

Reply:
Yeah, I'm happy with the trade-off here (having checked all the SR compstates they're fine with this change).